katcp/server.py

```diff
@@ -2325,7 +2326,7 @@ def set_log_level(self, level):
         """
         self._log_level = level
         if self._python_logger:
-            level = PYTHON_LEVEL.get(level)
+            level = self.PYTHON_LEVEL.get(level)
```

Comment:
Good point, but I'd beware of the setting it to NOTSET ... in such a case you will flood your logging system unintentionally and probably unknowingly. 

Rather just revert back to the default which is WARN.